src/components/rule-side-bar/rule-side-bar.js

```diff
@@ -16,6 +17,29 @@ const tooltipData = {
   },
 };
 
+const Contact = () => {
+  const [displayContactForm, setDisplayContactForm] = useState(false);
+
+  const onContactButtonClick = () => {
+    setDisplayContactForm(!displayContactForm);
+  };
+  return (
+    <div className="flex items-center justify-center">
+      <button
+        className="btn btn-red text-sm normalcase"
```

Comment:
Use built in tailwind class for this
https://tailwindcss.com/docs/text-transform

```suggestion
        className="btn btn-red text-sm normal-case"
```